easybuild/easyblocks/s/siesta.py

```diff
@@ -125,7 +126,7 @@ def configure_step(self):
         # Populate start_dir with makefiles
         run_cmd(os.path.join(start_dir, 'Src', 'obj_setup.sh'), log_all=True, simple=True, log_output=True)
 
-        if loose_ver < LooseVersion('4.1-b2'):
+        if loose_ver < LooseVersion('4.1.0.2'):
```

Comment:
shouldn't this be `4.1.0.22` then? (in case someone creates an easyconfig for `4.1-b1` -> `4.1.0.21`)

Reply:
Yeah it should, so I adjusted things a bit...